contribs/gmf/src/services/themesservice.js

```diff
@@ -103,24 +114,76 @@ gmf.Themes.findTheme_ = function(themes, themeName) {
  * @return {angular.$q.Promise} Promise.
  */
 gmf.Themes.prototype.getBgLayers = function() {
+
   goog.asserts.assert(!goog.isNull(this.promise_));
   return this.promise_.then(goog.bind(
       /**
        * @param {gmf.ThemesResponse} data The "themes" web service response.
-       * @return {Array.<Object>} Array of background layer objects.
+       * @return {angular.$q.Promise}
        */
       function(data) {
-        var bgLayers = data['background_layers'].map(goog.bind(function(item) {
-          goog.asserts.assert('name' in item);
-          goog.asserts.assert('imageType' in item);
+        var promises = data['background_layers'].map(goog.bind(function(item) {
```

Comment:
`Array.protoype.map` accepts a `thisArg` as second argument, maybe we can get rid of the use of `goog.bind` ?